docs/guides/javascript.md

```diff
@@ -23,6 +23,7 @@ spectral
 ```
 
 Please note that by default Spectral supports YAML 1.2 with merge keys extension.
+As recommended by the [spec](https://yaml.org/spec/1.2/spec.html#id2804923), tags are resolved according to the "Core schema".
```

Comment:
```suggestion
As recommended by the [spec](https://yaml.org/spec/1.2/spec.html#id2804923), tags are resolved according to the "Core Schema".
```

Reply:
Honestly though, YAML spec isn't clear about the schema :sweat_smile:.
https://medium.com/@P0lip/yaml-is-great-but-dont-use-it-2d02a60181ab section 6.